Configuration/TCA/tx_telephonedirectory_domain_model_languageskill.php

```diff
@@ -1,4 +1,28 @@
 <?php
+
+$languageSkills = [
+    [
+        'LLL:EXT:telephonedirectory/Resources/Private/Language/locallang_db.xlf:tx_telephonedirectory_domain_model_languageskill.skill.0',
```

Comment:
very bad idea. As TCA will be concatenated this variable $languageSkill will be available in all further TCA globally. Please...don't do that.